crates/router/src/core/admin.rs

```diff
@@ -2673,6 +2697,45 @@ pub async fn update_payment_connector(
         )
         .await?,
     };
+    #[cfg(all(feature = "v2", feature = "merchant_connector_account_v2"))]
+    let payment_connector = storage::MerchantConnectorAccountUpdate::Update {
+        merchant_id: None,
```

Comment:
This field should be deprecated, Merchant connector accounts merchant id we should not update.